integrations/operator/crdgen/testdata/protofiles/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2485,6 +2485,38 @@ message UpstreamInventoryHello {
   string ExternalUpgrader = 5;
   // ExternalUpgraderVersion identifies the external upgrader version. Empty if no upgrader is defined.
   string ExternalUpgraderVersion = 6;
+  // UpdaterV2Info is used by Teleport to send information about how the Teleport updater is doing.
```

Comment:
Did this accidentally sneak in btw? Doesn't seem related to your changes.

Reply:
This got pulled into the original PR too, due to `make -C crdgen update-snapshot` being out of date. I think its harmless? It's a step in the operator's `CONTRIBUTING.md` that I ran while debugging the test failure. Turned out not to be needed, but the snapshot was out of date, so maybe leaving it is for the best.